subprojects/inline/inline.gradle

```diff
@@ -0,0 +1,18 @@
+description = "Mockito preconfigured inline mock maker (intermediate and to be superseeded by automatic usage in a future version)"
+
+ext {
+    artifactName = 'mockito-inline'
+    bintrayAutoPublish = true
+    bintrayRepo = 'maven'
+    mavenCentralSync = false
+}
+
+apply from: "$rootDir/gradle/java-library.gradle"
+
+dependencies {
+    compile project.rootProject
```

Comment:
I checked and the created POM does contain the `mockito-core` dependency. Adding verification seems like a good idea!